src/games/strategy/triplea/ui/RouteOptimizer.java

```diff
@@ -105,7 +96,9 @@ private Point normalizePoint(Point point) {
    * @return A List of all possible Points depending in map Properties
    *         size may vary
    */
-  private List<Point2D> getPossiblePoints(Point2D point) {
+  public List<Point2D> getPossiblePoints(Point2D point) {
+    mapWidth = mapPanel.getImageWidth();
```

Comment:
A comment here that the map panel may have changed size could be good context to capture. That is my assumption about why were are updating these.